### DISCUSSION TITLE
Needs Status when loading

### DISCUSSION THREAD

**Discussion by GunTotnHippie** [vergecurrency/MyVergies#48] [Ideas]
Love the wallet!  This is the way a crypto wallet should be.  I only have one issue when first loading the wallet.  With a low bandwidth connection the wallet sits in a wait period for up to a few minutes.  I really think the app needs to be more verbose with its loading.  When I first load my wallet my heart stops waiting to make sure my total XVG shows up.  

"Connected to the Internet" || "Can not connect to the internet"
"Connecting to TOR"
"Generating TOR identity"
"Loading Wallet., .., ..., ...., ....." < with loading sequence so you know its still loading
"Checking/Loading Transactions 1/10, 2/10, 3/10"
"Completed"

Also it would be great to see the nodes you are connected to under settings or the ? icon

Otherwise great job on this wallet!  I can't wait to see one just like this for Android/IOS

**Comment by justinvforvendetta** [comment]
we did this.
